apps/coscrad-frontend/src/components/nav-menu/build-nav-menu-items.ts

```diff
@@ -0,0 +1,19 @@
+import { buildRoutes } from '../../app/build-routes';
+import { ID_ROUTE_PARAM_KEY } from '../../app/routes';
+import { ConfigurableContent } from '../../configurable-front-matter/data/configurable-content-schema';
+
+const unlistedRoutes = ['*'];
+
+export const buildNavMenuItems = (contentConfig: ConfigurableContent) => {
+    const routes = buildRoutes(contentConfig);
+
+    // TODO We should have a `shouldPublishInMenu` or `menus: string[]`  prop on CoscradRoute
```

Comment:
Let's deal with this next time we update the menu. Ideally, we would only tack on a flag in the case that we want to publish to the menu, but do nothing otherwise. Also, we should consider that we may have multiple menus later. So we might actually want a list of menus to publish to, one option being the 'NAV_MENU'.